packages/authentication/src/keys.ts

```diff
@@ -86,25 +86,6 @@ export namespace AuthenticationBindings {
     'authentication.operationMetadata',
   );
 
-  /**
-   * Key used to inject the user instance retrieved by the
-   * authentication function
-   *
-   * @example
-   * ```ts
-   * class MyController {
-   *   constructor(
-   *     @inject(AuthenticationBindings.CURRENT_USER) private user: UserProfile,
-   *   ) {}
-   *
-   * // ... routes that may need authentication
-   * ```
-   * }
-   */
-  export const CURRENT_USER = BindingKey.create<UserProfile | undefined>(
-    'authentication.currentUser',
-  );
-
```

Comment:
Let's keep `AuthenticationBindings.CURRENT_USER` as an alias to `SecurityBinding.CURRENT_USER`:

```ts
export const CURRENT_USER = SecurityBindings.CURRENT_USER;
```

Reply:
@jannyHou ^^^